subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -21,6 +21,33 @@ LOG_MODULE_REGISTER(net_wifi_mgmt, CONFIG_NET_L2_WIFI_MGMT_LOG_LEVEL);
 #include <zephyr/net/wifi_nm.h>
 #endif /* CONFIG_WIFI_NM */
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_ROAMING
+#define MAX_NEIGHBOR_AP_LIMIT 6U
+#define MAX_EVENT_STR_LEN 32U
+
+struct wifi_rrm_neighbor_ap_t {
+	char ssid[WIFI_SSID_MAX_LEN];
```

Comment:
Please allocate space for `'\0'`, so this needs to have +1

Reply:
Updated